pulsar-client/src/main/java/org/apache/pulsar/client/impl/UnAckedMessageTracker.java

```diff
@@ -150,7 +150,7 @@ public void run(Timeout t) throws Exception {
         }, this.tickDurationInMs, TimeUnit.MILLISECONDS);
     }
 
-    public static void addChunkedMessageIdsAndRemoveFromSequnceMap(MessageId messageId, Set<MessageId> messageIds,
+    public static void addChunkedMessageIdsAndRemoveFromSequenceMap(MessageId messageId, Set<MessageId> messageIds,
```

Comment:
I am not sure it is a good practice to change this unrelated code with this patch.
this patch is focused on LoadBalancer.
it is better to still to change things only in one module

Reply:
Ok, I will update later.